Source/Scene/LabelCollection.js

```diff
@@ -339,13 +437,21 @@ define([
         this._scene = options.scene;
 
         this._textureAtlas = undefined;
+        this.__backgroundTextureAtlas = undefined;
+        this._whitePixelIndex = undefined;
+
+        this._backgroundBillboardCollection = new BillboardCollection({
+            scene : this._scene
+        });
+        this._backgroundBillboardCollection.destroyTextureAtlas = false;
 
         this._billboardCollection = new BillboardCollection({
             scene : this._scene
         });
         this._billboardCollection.destroyTextureAtlas = false;
 
         this._spareBillboards = [];
+        this._spareBackgroundBillboards = [];
```

Comment:
Add/remove billboards causes the GL buffer to be recreated in the next `update` (so that is per frame, not per billboard).

The spares appear to be an "optimization" for setting show to true/false/true.  Probably not needed.